tools/create_release_notes.py

```diff
@@ -34,6 +35,7 @@
 RELEASE_FILE_CONTENT_TEMPLATE = """\
 ---
 title: OpenPOWER Host OS release
+release_tag: {release_tag}
```

Comment:
imho we do not need the "tag" word in all variables names here. The fact that the release is a git repository tag is irrelevant here, we should avoid unnecessary coupling.